resources/assets/css/coreui4.css

```diff
@@ -222,3 +222,11 @@ h1[bp-section=page-heading] {
     font-size: 2rem;
     line-height: 2.1rem;
 }
+
+html:not([dir=rtl]) .wrapper {
+    padding-left: var(--cui-sidebar-occupy-start, 0)
+}
```

Comment:
This seems very dangerous. Why are we changing the padding on ALL wrappers? Is there any way we can be more surgical here?

I suspect this is what triggered https://github.com/Laravel-Backpack/theme-coreuiv4/pull/24#issuecomment-1611690864

And it might trigger other issues too.

What are we fixing with this @karandatwani92 ?